compiler/dalgona/CMakeLists.txt

```diff
@@ -2,18 +2,44 @@
 #   Ubuntu18.04; explictly installed python3.8 (default is python3.6)
 #   Ubuntu20.04; default python3.8
 #   Ubuntu22.04; default python3.10
+#   Ubuntu24.04; explictly installed python3.8 (default is python3.12)
 #   refer https://github.com/Samsung/ONE/issues/9962
-find_package(PythonInterp 3.8 QUIET)
-find_package(PythonLibs 3.8 QUIET)
+if(CMAKE_VERSION VERSION_LESS 3.12)
```

Comment:
cmake versions for Ubuntu

![image](https://github.com/user-attachments/assets/4e76ed24-21c8-4e85-8087-74549ee2df1d)